Testscripts/Windows/RUNTIME_MEM_STRESSHOTREMOVE.ps1

```diff
@@ -150,6 +150,8 @@ function Main {
         Write-LogInfo "${vmName}: assigned - $vm1AfterAssigned | demand - $vm1AfterDemand"
         Write-LogInfo "Reported free memory inside ${vmName}: Before - $vm1BeforeAssignedGuest KB | After - $vm1AfterAssignedGuest KB"
         # get memory stats after stress-ng stopped running
+        # The duration of stress-ng is 120s, so wait any time bigger than it is OK, here 80 + 100 > 120
```

Comment:
I don't understand what it means.

Reply:
Sleep for a while to wait the stress-ng stopped running